### PR TITLE
Update with changes from Dev

### PR DESCRIPTION
**Pull request by budak7273**

### CODE REVIEW

**Comment by budak7273** [comment]
Closes #180

**Review by budak7273 — COMMENTED** [review]
Reviewed

File: modules/ROOT/pages/Development/Satisfactory/CustomLevels.adoc [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+Lumen is not on!::
+  Need resarch how to apply video settings to post process volume.
+* {blank}
++
+There is no stars in the night!::
+  Yes. You need to add stars somehow.
+* {blank}
++
+The sky is dark and I crash a few seconds after loading into the level!::
+  Make sure your Project Settings have FGWorldSettings as the World Settings Class.
```
Comment (mircearoata) [inline modules/ROOT/pages/Development/Satisfactory/CustomLevels.adoc:311]: Some casing issues/typos in this section

File: modules/ROOT/pages/ForUsers/ConfiguringMods.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
+= Configuring Mods
+
+There are two primary ways that mods offer configuration options:
+**Mod Configuration Options** and **Mod Savegame Settings**.
+
+== Mod Configuration Options
+
+Most using the SML Configuration System can have their configuration edited in-game
```
Comment (mircearoata) [inline modules/ROOT/pages/ForUsers/ConfiguringMods.adoc:8]:
```suggestion
Mods using the SML Configuration System can have their configuration edited in-game
```

File: modules/ROOT/pages/ForUsers/SatisfactoryModManager.adoc [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+Answers to frequent questions and troubleshooting steps for using the Mod Manager
+can be found on the xref:faq.adoc[Frequently Asked Questions] page.
+
+== Source Code
+
+// TODO SMMv3 update the language mentioned here
+The Mod Manager is written in Vue and is open source.
+You can find the source code on
+https://github.com/satisfactorymodding/SatisfactoryModLauncher/releases[GitHub]!
```
Comment (mircearoata) [inline modules/ROOT/pages/ForUsers/SatisfactoryModManager.adoc:191]:
```suggestion
https://github.com/satisfactorymodding/SatisfactoryModManager[GitHub]!
```

**Review by mircearoata — CHANGES_REQUESTED** [review]

**Review by mircearoata — APPROVED** [review]

**Pull request merged**